torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4427,18 +4426,19 @@ def _maybe_evaluate_static(
                 vr = self._default_unspecified_value_range()
             if size_oblivious and k in self.size_like:
                 lower = max(2, vr.lower)
+                upper = min(2 ** 48, vr.upper)
```

Comment:
This is responsible for ensuring that we can answer u0 != sys.maxsize. We just pick a "reasonable" upper bound for size-like unbacked symbols and assume they're below it. As long as you don't blow it up too much, things should work out.